.nvmrc

```diff
@@ -0,0 +1 @@
+v10.11.0
```

Comment:
Could we make this say 10 so that we get all the updates as well and don't have to keep bumping this?